YIF_Backend/Controllers/InstitutionOfEducationAdminController.cs

```diff
@@ -139,5 +139,24 @@ public async Task<IActionResult> GetIoEInfoByUserId()
             var result = await _ioEAdminService.GetIoEInfoByUserId(userId);
             return Ok(result.Object);
         }
+
+        /// <summary>
+        /// Get specialty description in IoE.
+        /// </summary>
+        /// <response code="200">Get full description of specialty in IoE</response>
+        /// <response code="400">If id is not valid.</response>
+        /// <response code="401">If user is unauthorized, token is bad/expired</response>
+        /// <response code="403">If user is not institution of education admin</response>
+        /// <response code="404">Specialty not found</response>
```

Comment:
There are comments for 400, 401, 403 status codes but there aren't attributes for them.

Reply:
Added attributes for codes 400, 401, 403.